syntax = "proto3";

// Import the TimeSpec.proto file
import "TimeSpec.proto";

message HeaderandWaveform {
  int32 apid = 1;
  int32 counter = 2;
  int32 type = 3;
  TimeSpec time = 4;
  int32 abstime = 5;
  int32 runID = 6;
  int32 configID = 7;
  int32 equalizationLevel = 8;
  int32 decimation = 9;
  int32 curOffset = 10;
  int32 trigOff = 11;
  int32 size = 12;
  repeated int32 data = 13;
  int32 crc = 14;
}
